### PR TITLE
opentitan: remove architecture reference from mainboard

### PR DESCRIPTION
**Pull request by rminnich**
we found in coreboot that maintaining a specific hierarchy of reference
made life best.

Currently we reference the architecture from both the mainboard and the
cpu. That gets messy, we have learned.

Reference the architecture from the cpu. Implement halt for the
cpu and the soc.

Use soc::halt in the mainboard code.

### CODE REVIEW

File: src/cpu/lowrisc/ibex/src/lib.rs [hunk 1]
```diff
@@ -4,3 +4,9 @@
 pub fn init() {
     arch::init()
 }
+
+// There is no ibex-specific way of halting yet.
+
+pub fn halt() -> ! {
```
Comment (rjoleary) [inline src/cpu/lowrisc/ibex/src/lib.rs]: Could you instead do:

```
pub use arch::halt
```

I think something like this is possible in rust?

Comment (rminnich) [reply src/cpu/lowrisc/ibex/src/lib.rs]: yes. done.

**Comment by lenary** [comment]
I'm working on tracing the CI failure you're seeing. Will update this patch when I'm further through the investigation.

**Comment by lenary** [comment]
That Rust pull request has been approved, so this issue might be fixed by the time this lands in nightly.

I expect this to have some fall-out within the project, as you will no longer have access to any `AtomicU*` or `AtomicI*` data types, which will need to be coded around.

**Comment by rminnich** [comment]
I did a rustup to 11/20 version and ... still doesn't work. But maybe it did not  percolate through yet.

**Comment by lenary** [comment]
Yeah, sadly I cannot see what's in the 20 November 2019 nightly edition (and https://rust-lang.github.io/rustup-components-history/ does not tell you which commit each nightly was built from)

**Comment by lenary** [comment]
I downloaded today's nightly (`rustc 1.41.0-nightly (f1b882b55 2019-11-20)`, for which the sha is the important part, and the nightly date was actually 2019-11-21).  I'm running on this on `x86_64-unknown-linux-gnu`.

I've attached the build output, but I think the summary is it's all successful (except a few warnings).

<details>
<summary>Build Output</summary>
```
[selliott@morty ~/oreboot/src/mainboard/opentitan/crb]$ cargo make -p release
[cargo-make] INFO - cargo make 0.23.0
[cargo-make] INFO - Project: crb
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: default
[cargo-make] INFO - Profile: release
[cargo-make] INFO - Running Task: empty
[cargo-make] INFO - Running Task: install-rust-src
[cargo-make] INFO - Running Task: build
[cargo-make] INFO - Execute Command: "rustup" "run" "nightly" "cargo" "xbuild" "--release" "--verbose"
+ "rustc" "--print" "sysroot"
+ "rustc" "--print" "target-list"
+ "/home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo" "rustc" "-p" "alloc" "--release" "--manifest-path" "/tmp/xargo.IIV3WCDXrFSS/Cargo.toml" "--target" "riscv32imc-unknown-none-elf" "-v" "--" "-Z" "force-unstable-if-unmarked"
    Updating crates.io index
   Compiling core v0.0.0 (/home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/src/libcore)
   Compiling compiler_builtins v0.1.21
     Running `rustc --edition=2018 --crate-name core /home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/src/libcore/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=fff4e0f023f464e5 -C extra-filename=-fff4e0f023f464e5 --out-dir /tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/release/deps -C link-arg=-Tlink.ld`
     Running `rustc --crate-name build_script_build /home/selliott/.cargo/registry/src/github.com-1ecc6299db9ec823/compiler_builtins-0.1.21/build.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C opt-level=z --cfg 'feature="compiler-builtins"' --cfg 'feature="core"' --cfg 'feature="default"' --cfg 'feature="mem"' -C metadata=dd222e072595fefe -C extra-filename=-dd222e072595fefe --out-dir /tmp/xargo.IIV3WCDXrFSS/target/release/build/compiler_builtins-dd222e072595fefe -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/release/deps --cap-lints allow`
     Running `/tmp/xargo.IIV3WCDXrFSS/target/release/build/compiler_builtins-dd222e072595fefe/build-script-build`
   Compiling rustc-std-workspace-core v1.99.0 (/home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/src/tools/rustc-std-workspace-core)
     Running `rustc --edition=2018 --crate-name rustc_std_workspace_core /home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/src/tools/rustc-std-workspace-core/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=6b7922c199807f0a -C extra-filename=-6b7922c199807f0a --out-dir /tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/release/deps --extern core=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps/libcore-fff4e0f023f464e5.rmeta -C link-arg=-Tlink.ld`
     Running `rustc --crate-name compiler_builtins /home/selliott/.cargo/registry/src/github.com-1ecc6299db9ec823/compiler_builtins-0.1.21/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z --cfg 'feature="compiler-builtins"' --cfg 'feature="core"' --cfg 'feature="default"' --cfg 'feature="mem"' -C metadata=1b7ba1a6fb9fa027 -C extra-filename=-1b7ba1a6fb9fa027 --out-dir /tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/release/deps --extern core=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps/librustc_std_workspace_core-6b7922c199807f0a.rmeta --cap-lints allow -C link-arg=-Tlink.ld --cfg 'feature="unstable"'`
   Compiling alloc v0.0.0 (/tmp/xargo.IIV3WCDXrFSS)
     Running `rustc --edition=2018 --crate-name alloc /home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/src/liballoc/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -Z force-unstable-if-unmarked -C metadata=4a4e64e8cae2f8df -C extra-filename=-4a4e64e8cae2f8df --out-dir /tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/tmp/xargo.IIV3WCDXrFSS/target/release/deps --extern compiler_builtins=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps/libcompiler_builtins-1b7ba1a6fb9fa027.rmeta --extern core=/tmp/xargo.IIV3WCDXrFSS/target/riscv32imc-unknown-none-elf/release/deps/libcore-fff4e0f023f464e5.rmeta -C link-arg=-Tlink.ld`
    Finished release [optimized] target(s) in 7.48s
+ RUSTFLAGS="-C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot"
+ RUSTDOCFLAGS="-C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot"
+ "/home/selliott/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo" "build" "--release" "--verbose"
   Compiling model v0.1.0 (/home/selliott/oreboot/src/drivers/model)
   Compiling tock-registers v0.3.0
     Running `rustc --edition=2018 --crate-name model /home/selliott/oreboot/src/drivers/model/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=2a93dbcc156f20f3 -C extra-filename=-2a93dbcc156f20f3 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
   Compiling clock v0.1.0 (/home/selliott/oreboot/src/drivers/clock)
   Compiling arch v0.1.0 (/home/selliott/oreboot/src/arch/riscv/rv32)
   Compiling crb v0.1.0 (/home/selliott/oreboot/src/mainboard/opentitan/crb)
   Compiling console v0.1.0 (/home/selliott/oreboot/src/console)
   Compiling static-ref v0.1.1
     Running `rustc --edition=2018 --crate-name tock_registers /home/selliott/.cargo/registry/src/github.com-1ecc6299db9ec823/tock-registers-0.3.0/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=fa0ae33516dd410a -C extra-filename=-fa0ae33516dd410a --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --cap-lints allow -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --edition=2018 --crate-name clock /home/selliott/oreboot/src/drivers/clock/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=b5b8c3d50f1a1cfc -C extra-filename=-b5b8c3d50f1a1cfc --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --edition=2018 --crate-name arch /home/selliott/oreboot/src/arch/riscv/rv32/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=f24899bdc40f569f -C extra-filename=-f24899bdc40f569f --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --edition=2018 --crate-name build_script_build build.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C opt-level=z -C metadata=df8b59ae9970d5f1 -C extra-filename=-df8b59ae9970d5f1 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/build/crb-df8b59ae9970d5f1 -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps`
     Running `rustc --edition=2018 --crate-name console /home/selliott/oreboot/src/console/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=979e69a8927b7f0e -C extra-filename=-979e69a8927b7f0e --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --crate-name static_ref /home/selliott/.cargo/registry/src/github.com-1ecc6299db9ec823/static-ref-0.1.1/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=d4afd8ca37860c40 -C extra-filename=-d4afd8ca37860c40 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --cap-lints allow -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
   Compiling cpu v0.1.0 (/home/selliott/oreboot/src/cpu/lowrisc/ibex)
     Running `rustc --edition=2018 --crate-name cpu /home/selliott/oreboot/src/cpu/lowrisc/ibex/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=a0f4a232ac322f15 -C extra-filename=-a0f4a232ac322f15 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern arch=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libarch-f24899bdc40f569f.rmeta -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
   Compiling payloads v0.1.0 (/home/selliott/oreboot/payloads)
   Compiling wrappers v0.1.0 (/home/selliott/oreboot/src/drivers/wrappers)
   Compiling print v0.1.0 (/home/selliott/oreboot/src/lib/print)
     Running `rustc --edition=2018 --crate-name payloads /home/selliott/oreboot/payloads/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=b527c479a5a2f83b -C extra-filename=-b527c479a5a2f83b --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern model=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libmodel-2a93dbcc156f20f3.rmeta -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --edition=2018 --crate-name wrappers /home/selliott/oreboot/src/drivers/wrappers/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=193663a2304201b6 -C extra-filename=-193663a2304201b6 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern model=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libmodel-2a93dbcc156f20f3.rmeta -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --edition=2018 --crate-name print /home/selliott/oreboot/src/lib/print/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=5779740b59d0ea40 -C extra-filename=-5779740b59d0ea40 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern model=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libmodel-2a93dbcc156f20f3.rmeta -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
   Compiling register v0.3.3
     Running `rustc --edition=2018 --crate-name register /home/selliott/.cargo/registry/src/github.com-1ecc6299db9ec823/register-0.3.3/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=c39a3f7525ae00c5 -C extra-filename=-c39a3f7525ae00c5 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern tock_registers=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libtock_registers-fa0ae33516dd410a.rmeta --cap-lints allow -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
   Compiling soc v0.1.0 (/home/selliott/oreboot/src/soc/opentitan/earlgrey)
   Compiling uart v0.1.0 (/home/selliott/oreboot/src/drivers/uart)
     Running `rustc --edition=2018 --crate-name soc /home/selliott/oreboot/src/soc/opentitan/earlgrey/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z -C metadata=234a3798542e4af4 -C extra-filename=-234a3798542e4af4 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern clock=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libclock-b5b8c3d50f1a1cfc.rmeta --extern cpu=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libcpu-a0f4a232ac322f15.rmeta --extern model=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libmodel-2a93dbcc156f20f3.rmeta --extern register=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libregister-c39a3f7525ae00c5.rmeta -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `rustc --edition=2018 --crate-name uart /home/selliott/oreboot/src/drivers/uart/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=z --cfg 'feature="sifive"' -C metadata=4cdeca402b1b4570 -C extra-filename=-4cdeca402b1b4570 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern clock=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libclock-b5b8c3d50f1a1cfc.rmeta --extern model=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libmodel-2a93dbcc156f20f3.rmeta --extern register=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libregister-c39a3f7525ae00c5.rmeta -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
     Running `/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/build/crb-df8b59ae9970d5f1/build-script-build`
     Running `rustc --edition=2018 --crate-name crb src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C opt-level=z -C lto -C metadata=771376b43db38a86 -C extra-filename=-771376b43db38a86 --out-dir /home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps --target riscv32imc-unknown-none-elf -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps -L dependency=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/release/deps --extern console=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libconsole-979e69a8927b7f0e.rlib --extern model=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libmodel-2a93dbcc156f20f3.rlib --extern payloads=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libpayloads-b527c479a5a2f83b.rlib --extern print=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libprint-5779740b59d0ea40.rlib --extern register=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libregister-c39a3f7525ae00c5.rlib --extern soc=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libsoc-234a3798542e4af4.rlib --extern static_ref=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libstatic_ref-d4afd8ca37860c40.rlib --extern uart=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libuart-4cdeca402b1b4570.rlib --extern wrappers=/home/selliott/oreboot/src/mainboard/opentitan/crb/target/riscv32imc-unknown-none-elf/release/deps/libwrappers-193663a2304201b6.rlib -C link-arg=-Tlink.ld --sysroot /home/selliott/oreboot/src/mainboard/opentitan/crb/target/sysroot`
    Finished release [optimized] target(s) in 0.39s
[cargo-make] INFO - Running Task: bootblob
[cargo-make] INFO - Execute Command: "cargo" "objcopy" "--" "-O" "binary" "-R" ".bss" "target/riscv32imc-unknown-none-elf/release/crb" "target/riscv32imc-unknown-none-elf/release/bootblob.bin"
[cargo-make] INFO - Running Task: default
fixed-dtfs.dts:13.20-18.15: Warning (unit_address_vs_reg): /flash-info/areas/area@0: node has a unit name, but no reg property
fixed-dtfs.dts:19.20-24.15: Warning (unit_address_vs_reg): /flash-info/areas/area@1: node has a unit name, but no reg property
fixed-dtfs.dts:25.20-30.15: Warning (unit_address_vs_reg): /flash-info/areas/area@2: node has a unit name, but no reg property
fixed-dtfs.dts:31.20-36.15: Warning (unit_address_vs_reg): /flash-info/areas/area@3: node has a unit name, but no reg property
fixed-dtfs.dts:37.20-42.15: Warning (unit_address_vs_reg): /flash-info/areas/area@4: node has a unit name, but no reg property
fixed-dtfs.dts:43.20-47.15: Warning (unit_address_vs_reg): /flash-info/areas/area@5: node has a unit name, but no reg property
[cargo-make] INFO - Running Task: empty
[cargo-make] INFO - Build Done in 8 seconds.
```
</details>

**Comment by rminnich** [comment]
already done in another PR

**Pull request closed without merging**